app/src/utils/PremiumUtils.js

```diff
@@ -22,6 +22,8 @@ export const getPlanNameFromId = (planId) => {
       return PRICING.PLAN_NAMES.API_CLIENT_PROFESSIONAL;
     case PRICING.PLAN_NAMES.API_CLIENT_ENTERPRISE:
       return PRICING.PLAN_NAMES.API_CLIENT_ENTERPRISE;
+    case PRICING.PLAN_NAMES.PROFESSIONAL_STUDENT:
+      return PRICING.PLAN_NAMES.PROFESSIONAL_STUDENT;
```

Comment:
No need for this change. planName is matched by splitting planId by _ which will be professional.